### PR TITLE
Html based Ver 1.0.4 

### PR DESCRIPTION
**Pull request by boefraty74**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "visual",
   "scripts": {
-    "postinstall": "pbiviz update 1.7.0",
+    "postinstall": "pbiviz update 1.10.0",
```
Comment (ignatvilesov) [inline package.json:4]: Please remove `package-lock.json` from git

File: package.json [hunk 2]
```diff
@@ -10,6 +10,6 @@
   "devDependencies": {
     "tslint": "^4.4.2",
     "tslint-microsoft-contrib": "^4.0.0",
-    "powerbi-visuals-tools": "1.7.1"
+    "powerbi-visuals-tools": "1.10.0"
```
Comment (ignatvilesov) [inline package.json]: As you use powerbi-visuals-utils-dataview-tuils you would need to add `"powerbi-visuals-utils-dataviewutils": "1.2.0"` as well.

**Review by ignatvilesov — CHANGES_REQUESTED** [review]
Please also fix all of TSLint issues reported by Travis CI

https://travis-ci.org/Microsoft/PowerBI-visuals-forcasting-exp/builds/370931675?utm_source=github_status&utm_medium=notification

File: package.json (hunk 1 above)
Comment (boefraty74) [reply package.json:4]: done

**Review by ignatvilesov — APPROVED** [review]

**Pull request merged**